### PR TITLE
weeknotes1(v01), weeknotes2(v02), weeknotes3 added

### PR DESCRIPTION
**Pull request by renganathanrc**
Hi brother, week notes 3 added. Kindly add your comments.

### CODE REVIEW

File: content/blog/2020/Weeknotes3.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Week3-20200607"
+date: 2020-06-15T19:08:09-04:00
+categories: ["RIT","Weeknotes","Web Development"]
+---
+As mentioned in the previous week, I learned the basics of sqlite database using the website and simultaneously created the basic todo webapp using the Youtube tutorial.
+### **SQLite learning process**
+The website uses a good strategy **(Spaced Repetition system)** to teach the required content and only 16 lessons are free. So it takes the whole week to progress through the free lessons.
+>See previous weeknote for the site's link
```
Comment (amjith) [inline content/blog/2020/Weeknotes3.md:9]: Don't make it hard for your readers. It doesn't cost you anything to add a link to the site here. 

Also make each weeknote self-contained. Don't make it a continuation of the previous note. 

For instance, don't assume that someone is going to read your weeknotes in order.

File: content/blog/2020/Weeknotes3.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    * Create a python script that used Flask to display a basic message
+3. Templates and static content
+    * Create the basic templates and static contents to render your web app
+4. Setting up and using Database (SQLite)
+    * Add necessary codes in the python script to setting up and interact with the database
+5. Create the basic todo app
+    * Add the routes and necessary codes that do the required actions (saving and retrieving) to create a fully functional basic todo app
+6. Deploy to Heroku
+    * After tested the above app in the local machine, add Heroku credentials and create the app and push the files into the Heroku account.
+>Again, for the tutorial link refer the previous weeknote. In this process, I faced many unknown terms in the flask code. Terms like classes, methods, and forms required a lot of extra understanding. For that I used W3Schools.com and GeeksforGeeks.com.
```
Comment (amjith) [inline content/blog/2020/Weeknotes3.md:29]: Same as above. Include the link here.

File: content/blog/2020/Weeknotes3.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+title: "Week3-20200607"
+date: 2020-06-15T19:08:09-04:00
+categories: ["RIT","Weeknotes","Web Development"]
+---
+As mentioned in the previous week, I learned the basics of sqlite database using the website and simultaneously created the basic todo webapp using the Youtube tutorial.
+### **SQLite learning process**
+The website uses a good strategy **(Spaced Repetition system)** to teach the required content and only 16 lessons are free. So it takes the whole week to progress through the free lessons.
+>See previous weeknote for the site's link
+### **Basic todo webapp**
+These are the steps used in the tutorial.
```
Comment (amjith) [inline content/blog/2020/Weeknotes3.md:11]: Give a brief description of this project. Say that it is a flask web app with an SQLite database. 

Say a few words about the features of the application you've created. 

If this is deployed then include a link to the app so the users can try it out themselves. 

Remember "Kanden Seethaiyai" attitude when you're writing the blog post. Start with the most important things.

**Comment by renganathanrc** [comment]
Changes made and commited

**Pull request closed without merging**